src/index.js

```diff
@@ -194,8 +195,61 @@ const HMMediaWeightSidebar = () => {
 		};
 	} );
 
+	const insertIframe = ( url, width = '1600px', height = '800px' ) => {
+		const iframeExists = document.getElementById( 'post-preview-iframe' );
```

Comment:
When content changes, and we maybe want to trigger this pre-publish, is it possible to refresh/update the iframe? I just had it adding a new frame each time, but if there's a way to reuse and trigger a fresh page load on one then this seems sensible.